plugins/module_utils/network/sonic/config/copp/copp.py

```diff
@@ -169,8 +170,8 @@ def set_config(self, existing_copp_facts):
         :returns: the commands necessary to migrate the current configuration
                   to the desired configuration
         """
-        want = self._module.params['config']
-        have = existing_copp_facts
+        want = remove_empties(self._module.params['config'])
+        have = remove_empties(existing_copp_facts)
```

Comment:
Since remove_empties is now being done during facts collection, is there a reason why it needs to be done here?

If not, please avoid the "remove_empties" invocation in the line above for "existing_copp_facts".